### ISSUE TITLE
Always getting Error 10114 nonce Small when trying to get trades_history

### ISSUE THREAD

**Issue by nhondong** [ohenrik/bitfinex#18]
Hi Henrik, 

I try to use the function trades_history from restv2, but I always got: 

> Traceback (most recent call last):
>   File "<input>", line 21, in <module>
>   File "/home/nhondong/PycharmProjects/venv/lib/python3.6/site-packages/bitfinex/rest/restv2.py", line 330, in trades_history
>     response = self._post(path, raw_body, verify=True)
>   File "/home/nhondong/PycharmProjects/venv/lib/python3.6/site-packages/bitfinex/rest/restv2.py", line 89, in _post
>     raise BitfinexException(response.status_code, response.reason, content)
> bitfinex.rest.restv2.BitfinexException: (500, 'Internal Server Error', ['error', 10114, 'nonce: small'])

```
from bitfinex.rest.restv2 import Client as Client2
key = config.bitfinex_apikey
secret = config.bitfinex_secret
BTFXCLIENT = Client2(key, secret)
TRADES = BTFXCLIENT.trades_history('tIOTUSD', limit=10) 
print(trades)
```
And an example for setting start and end would be nice.

Kind regards, 

Nils

**Comment by nhondong** [comment]
Thanks, I will try this.

**Comment by ohenrik** [comment]
@dantimofte is correct, and we should unify this for rest and websockets. To clarify, the nonce needs to be an increasing unique number within each day. Let us know if using another key works so we can close the issue :)

**Comment by dantimofte** [comment]
This has been fixed with the release of version 1.0.0 https://github.com/ohenrik/bitfinex/commit/fd13900a8b692aa013606836c58bf8f31d4079f1
